test/algorithms/xiao2013_utils.jl

```diff
@@ -0,0 +1,161 @@
+using EliminateGraphs, EliminateGraphs.Graphs
```

Comment:
I added some new tests, I think the algorithm is correctly implemented. Good job!